test/runtime/write.c

```diff
@@ -617,6 +618,59 @@ static void usage(const char *program_name)
            p);
 }
 
+static void fs_stress_test() 
+{
+    int num_files = 1000;
+    int fds[num_files];
+
+    /* Creating and writing to new files */
+    int i;
+    for (i = 0; i < num_files; i++) {
+        char buf[BUFLEN];
+        ssize_t rv;
+        char name[25];
+        sprintf(name, "fs_stress_test_%d", i);
+        int fd = open(name, O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
+        fds[i] = fd;
+        if (fd < 0) {
+            printf("open error\n");
+            perror("open");
+            exit(EXIT_FAILURE);
+        }
+
+        _READ(buf, BUFLEN);
+
+        if (rv == 0)
+            writetest_debug("empty source file\n");
```

Comment:
This writetest_debug() call is not really useful, since all files will be empty and this would just pollute the debug logs with 1000 identical lines